src/algebra/star/basic.lean

```diff
@@ -126,6 +129,29 @@ class star_semigroup (R : Type u) [semigroup R] extends has_involutive_star R :=
 export star_semigroup (star_mul)
 attribute [simp] star_mul
 
+section star_semigroup
+variables [semigroup R] [star_semigroup R]
+
+lemma star_star_mul (x y : R) : star (star x * y) = star y * x := by rw [star_mul, star_star]
+
+lemma star_mul_star (x y : R) : star (x * star y) = y * star x := by rw [star_mul, star_star]
+
+@[simp] lemma semiconj_by_star_star_star {x y z : R} :
+  semiconj_by (star x) (star z) (star y) ↔ semiconj_by x y z :=
+by simp_rw [semiconj_by, ←star_mul, star_inj, eq_comm]
+
+alias semiconj_by_star_star_star ↔ _ semiconj_by.star_star_star
+
+@[simp] lemma commute_star_star {x y : R} : commute (star x) (star y) ↔ commute x y :=
+semiconj_by_star_star_star
+
+alias commute_star_star ↔ _ commute.star_star
```

Comment:
Do you think the `of_` aliases are detrimental?

Reply:
I think they're pretty worthless here, though I can add them if you feel strongly.